react-app/hooks/redux.ts

```diff
@@ -1,5 +1,5 @@
-import { TypedUseSelectorHook, useDispatch, useSelector } from "react-redux";
-import { AppDispatch, RootState } from "../store/store";
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../store/store';
 
 export const useAppDispatch = () => useDispatch<AppDispatch>();
```

Comment:
[looks like this usAppDispatch hook will not work](https://redux.js.org/usage/usage-with-typescript#define-typed-hooks)

```suggestion
export const useAppDispatch: AppDispatch =  useDispatch;
```